app/views/works/index.html.erb

```diff
@@ -0,0 +1,88 @@
+<h2>List of Works</h2>
+
+<section class = "media-table">
+  <h4>Albums</h4>
+  <table>
+    <thead>
+      <tr>
+        <th>Votes</th>
+        <th>Title</th>
+        <th>Created By</th>
+        <th>Published</th>
+        <th>Upvote</th>
+      </tr>
+    </thead>
+
+    <tbody>
+
+      <tr>
+        <% @all_albums.each do |album| %>
+        <td><%= album.votes.count %></td>
+        <td><%= link_to album.title, work_path(album) %></td>
+        <td><%= album.creator %></td>
+        <td><%= album.publication_year %></td>
+        <td><%= link_to "Upvote", upvote_work_path(album), method: :post %></td>
+      </tr>
+      <% end %>
+
+    </tbody>
+  </table>
+</section>
+
+<section class = "media-table">
+  <h4>Books</h4>
+  <table>
+    <thead>
```

Comment:
You have the same code to show a list of works repeated 3 times. Could you use a view partial or a loop to DRY this up?